askama/Cargo.toml

```diff
@@ -22,12 +22,15 @@ default = []
 serde-json = ["askama_shared/serde-json"]
 with-iron = ["iron", "askama_derive/iron"]
 with-rocket = ["rocket", "askama_derive/rocket"]
+with-actix-web = ["actix-web", "askama_derive/actix-web", "mime_guess"]
 
 [dependencies]
 askama_derive = { path = "../askama_derive", version = "0.7.0" }
 askama_shared = { version = "0.7.0", path = "../askama_shared" }
 iron = { version = ">= 0.5, < 0.7", optional = true }
 rocket = { version = "0.3", optional = true }
+actix-web = { version = ">= 0.6", optional = true }
+mime_guess = { version = "2.0.0-alpha", optional = true }
```

Comment:
Can we depend on a stable version here, please?

Reply:
Ah, this `2.0.0-alpha`? That's...

- Unfortunately, what `actix-web` currently relies on
- And currently a choice of theirs to remain at this release until the `mime` crate stabilizes: https://github.com/abonander/mime_guess

I know, I found it weird too.